core/sail/lmdb/pom.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.rdf4j</groupId>
+		<artifactId>rdf4j-sail</artifactId>
+		<version>4.0.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>rdf4j-sail-lmdb</artifactId>
+	<name>RDF4J: LmdbStore</name>
+	<description>Sail implementation that stores data to disk using LMDB.</description>
+	<properties>
+		<lwjgl.version>3.3.0</lwjgl.version>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.lwjgl</groupId>
+			<artifactId>lwjgl-lmdb</artifactId>
+			<version>${lwjgl.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.lwjgl</groupId>
+			<artifactId>lwjgl-lmdb</artifactId>
+			<classifier>natives-linux</classifier>
+			<version>${lwjgl.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.lwjgl</groupId>
+			<artifactId>lwjgl-lmdb</artifactId>
+			<classifier>natives-macos-arm64</classifier>
+			<version>${lwjgl.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.lwjgl</groupId>
+			<artifactId>lwjgl-lmdb</artifactId>
+			<classifier>natives-macos</classifier>
+			<version>${lwjgl.version}</version>
+		</dependency>
```

Comment:
Can we make the various natives extensions optional dependencies perhaps? It seems a bit wasteful to force downloading all of them when any particular deployment will only need one. Also thinking about the size of our fat jars :)

Reply:
Yes, I think this is possible. How are optional dependencies handled in unit tests?

Reply:
That should not make a difference. An optional dependency is still available for us. It just means that any project that depends on our module does not automatically include them, transitively.